cmd/login/clientcert.go

```diff
@@ -139,22 +157,100 @@ func fetchCredential(ctx context.Context, provider string, clientCertService cli
 	return secret, nil
 }
 
+func generateCredential(ctx context.Context, ca *corev1.Secret, config clientCertConfig) (*corev1.Secret, error) {
```

Comment:
This is the function we use to create the client certificate for openstack clusters. We sign it using the CA secret for the WC.